_includes/technology/repos.md

```diff
@@ -1,21 +1,19 @@
 ## Repositories
 
-There are many individual repositories that make up eRegulations; this is
-complicated by having two primary forks maintained by CFPB and 18F
-respectively. While these forks will ultimately be merged and maintenance
-shared between multiple agencies, those interested in using eRegs now should
-generally prefer the 18F repositories.
+There are many individual repositories that make up eRegulations. The project is divided into several components, and currently CFPB and 18F maintain two separate forks of **regulations-parser**, **regulations-core**, and **regulations-site**. 18F and CFPB plan to eventually merge these forks into shared components with maintenance shared between multiple agencies. For now, in most cases people interested in using eRegulations should use the 18F repositories (listed under "General Purpose").
+
+*Tip: If you'd simply like to get a copy of eRegulations set up to try it out, try starting with [CFPB **regulations-bootstrap**](https://github.com/cfpb/regulations-bootstrap) or [ATF **atf-eregs**](https://github.com/18F/atf-eregs), which are both also listed below.*
```

Comment:
OK! My thinking was to integrate [this "getting started" bit from the CFPB eRegulations page](http://cfpb.github.io/eRegulations/#getting-started) for people who want a quick demo to mess with.

Could it make sense to say "If you'd like to get an example eRegulations site set up to try it out, check out [CFPB **regulations-bootstrap**](https://github.com/cfpb/regulations-bootstrap) for a quick install of a CFPB-centric version." - or better to just skip this for now?